packages/wrangler/src/__tests__/deployments.test.ts

```diff
@@ -247,7 +277,12 @@ describe("deployments", () => {
 					result: true,
 				});
 
-				await runWrangler("deployments rollback 3mEgaU1T-Intrepid-someThing");
+				mockPrompt({
+					text: "Please provide a message for this rollback (280 characters max)",
```

Comment:
i think we agreed on 120 char limit
```suggestion
					text: "Please provide a message for this rollback (120 characters max)",
```